hrqb/base/task.py

```diff
@@ -205,6 +213,12 @@ def target(self) -> QuickbaseTableTarget:
             table_name=self.table_name,
         )
 
+    @property
+    def parse_upsert_counts(self) -> dict | None:
+        if self.target.exists():
+            return QBClient.parse_upsert_results(self.target.read())
+        return None
+
```

Comment:
A docstring might help to clarify this property

Reply:
Added!  Good call.